src/lib/reasoners/uf.ml

```diff
@@ -553,7 +553,7 @@ module Env = struct
       ) env (X.leaves mkr)
 
   let init_term env t =
-    let mkr, ctx = X.make t in
+    let mkr, ctx = X.make_semantic t in
     let rp, ex = normal_form env mkr in
     let env =
       {env with
```

Comment:
There are other few (at least one) usage of X.make in Uf.ml that could be replaced with X.make_semantic

Reply:
The same remark applies for usage of X.make in intervalCalculus.ml and ccx.ml